scripts/qa_crowbarsetup.sh

```diff
@@ -2368,7 +2368,9 @@ function enable_ssl_generic
         ;;
     esac
     $p "$a['ssl']['generate_certs']" true
-    $p "$a['ssl']['insecure']" true
+    if [ "$service" != "rabbitmq" ] ; then
```

Comment:
maybe we should still add an insecure flag for rabbitmq, even if it is not used atm?